tap_aggregator/Cargo.toml

```diff
@@ -12,7 +12,7 @@ name = "tap_aggregator"
 path = "src/main.rs"
 
 [dependencies]
-tap_core = { path = "../tap_core", version = "2.0.0" }
+tap_core = { path = "../tap_core", version = "2.0.0", features = ["test"] }
```

Comment:
Not really liking this feature here in tap_aggregator

Reply:
Yeah... Why is it required?